code/game/objects/structures/aliens.dm

```diff
@@ -126,6 +126,8 @@
 	canSmoothWith = list(/obj/structure/alien/weeds, /turf/closed/wall)
 	smooth = SMOOTH_MORE
 
+var/list/blacklisted_weed_turfs = typecacheof(list(/turf/open/floor/plating/lava))
```

Comment:
@coiax 
1. typecacheof is a proc, so you could only put the base list there
2. lazy init of lists
3. Hidden ```Init()```

putting a list in the definition adds code to the hidden ```Init()``` proc, this proc only exists to handle certain (eg: list) proc calls done in the def, if you don't have anything that needs ```Init()```, it never happens. Since ```New()``` is always called, it's cheaper to put the code there, rather than force an ```Init()``` proc to exist.